couchdb/tests/client.py

```diff
@@ -374,6 +374,76 @@ def test_query_multi_get(self):
         for idx, i in enumerate(range(1, 6, 2)):
             self.assertEqual(i, res[idx].key)
 
+    def test_find(self):
+        if self.server.major() < 2:
```

Comment:
Ah, that's just sad. Can you maybe add a comment to these return statements saying that this is intended to skip the tests, but this is not available on 2.6?